### PR TITLE
Update generated files and examples

### PR DESCRIPTION
**Pull request by bertrand-lorentz**
Large batch of updates to the generated files, and to the examples to adapt for those changes.
This will be included in the upcoming 1.2.0 release.

You're probably better off looking at individual commits, and not the whole diff, as its quite large.

### CODE REVIEW

**Review by rouschr — APPROVED** [review]
timeperiod stuff looks so Ok for me

File: fields/fields.json [hunk 1]
```diff
@@ -2,8 +2,8 @@
   "ublVersion" : "2.3",
   "sdkVersion" : "eforms-sdk-1.1.0",
   "metadataDatabase" : {
-    "version" : "1.1.0",
-    "createdOn" : "2022-09-14T16:05:55"
+    "version" : "1.1.12",
```
Comment (rousso) [inline fields/fields.json:5]: I think we should go to  MDD version 1.2.0

File: schematrons/dynamic/complete-validation.sch [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8" ?>
-<!--File generated from metadata database version 1.1.0 created on the 2022-09-14T16:05:55.-->
+<!--File generated from metadata database version 1.1.14 created on the 2022-09-21T13:34:47.-->
```
Comment (rousso) [inline schematrons/dynamic/complete-validation.sch]: This database version (!.1.14) is different from the version used to create fields.json (1.1.12)

File: schematrons/dynamic/validation-stage-2a.sch [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" ?>
-<!--File generated from metadata database version 1.1.12 created on the 2022-09-20T17:20:53.-->
+<!--File generated from metadata database version 1.1.20 created on the 2022-09-21T17:25:19.-->
```
Comment (rousso) [inline schematrons/dynamic/validation-stage-2a.sch:2]: 1.1.20 != 1.1.14 != 1.1.12

**Review by rousso — CHANGES_REQUESTED** [review]
Apart from the inconsistencies in the database version used to generate everything (some of which I pointed out in my comments), I do not see any other issue. 

I understand that this had to be an iterative process and I assume that we will eventually bump the MDD version to 1.2.0 and regenerate everything with MDC in one go.

**Comment by bertrand-lorentz** [comment]
The metadata database version will be indeed updated to 1.2.0 for all files as part of the release branch.

**Pull request merged**